Hi.

This patch removes some debugging code that was using the block dump code to help diagnose some recent issues. I believe it's not needed anymore.

 io.c |    5 -----
 1 file changed, 5 deletions(-)
diff -ruNp 9190-remove-block-dump.patch-old/kernel/power/io.c 9190-remove-block-dump.patch-new/kernel/power/io.c
--- 9190-remove-block-dump.patch-old/kernel/power/io.c	2006-01-10 13:19:40.000000000 +1000
+++ 9190-remove-block-dump.patch-new/kernel/power/io.c	2006-01-10 13:04:01.000000000 +1000
@@ -931,18 +931,13 @@ out_reset_console:
  * Description:	Attempt to read the header and pageset1 of a suspend image.
  * 		Handle the outcome, complaining where appropriate.
  */
-extern int block_dump;
 
 int read_pageset1(void)
 {
 	int error;
 
-	block_dump = 1;
-
 	error = __read_pageset1();
 
-	block_dump = 0;
-	
 	switch (error) {
 		case 0:
 		case -ENODATA:

_______________________________________________
Suspend2-devel mailing list
Suspend2-devel@lists.suspend2.net
http://lists.suspend2.net/mailman/listinfo/suspend2-devel

